backend/tenant_account/migrations/0002_organizationmember_delete_user.py

```diff
@@ -13,33 +10,34 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.CreateModel(
-            name="OrganizationMember",
-            fields=[
-                (
-                    "user_ptr",
-                    models.OneToOneField(
-                        auto_created=True,
-                        on_delete=django.db.models.deletion.CASCADE,
-                        parent_link=True,
-                        primary_key=True,
-                        serialize=False,
-                        to=settings.AUTH_USER_MODEL,
-                    ),
-                ),
-                ("role", models.CharField(default="admin")),
-            ],
-            options={
-                "verbose_name": "user",
-                "verbose_name_plural": "users",
-                "abstract": False,
-            },
-            bases=("account.user",),
-            managers=[
-                ("objects", django.contrib.auth.models.UserManager()),
-            ],
-        ),
-        migrations.DeleteModel(
-            name="User",
-        ),
+        # migrations.CreateModel(
+        #     name="OrganizationMember",
+        #     fields=[
+        #         (
+        #             "user_ptr",
+        #             models.OneToOneField(
+        #                 auto_created=True,
+        #                 on_delete=django.db.models.deletion.CASCADE,
+        #                 parent_link=True,
+        #                 primary_key=True,
+        #                 serialize=False,
+        #                 to=settings.AUTH_USER_MODEL,
+        #             ),
+        #         ),
+        #         ("role", models.CharField(default="admin")),
+        #     ],
+        #     options={
+        #         "verbose_name": "user",
+        #         "verbose_name_plural": "users",
+        #         "abstract": False,
+        #     },
+        #     bases=("account.user",),
+        #     managers=[
+        #         ("objects", django.contrib.auth.models.UserManager()),
+        #     ],
+        # ),
+        # # commenting drop table to ignore AccesExclusive Lock
+        # migrations.DeleteModel(
+        #     name="User",
+        # ),
```

Comment:
@johnyrahul won't it be enough to just comment the drop table part alone? I see your changes on the previous migration but does it capture other details mentioned in options / managers?

Reply:
This migration was created as part of the name change of the model, we don't require both tables. Previous migrations have options/managers. can you please expand and validate